docs/source/standard/item.rst

```diff
@@ -21,26 +21,29 @@ Schema
     A description of the goods, services to be provided.
```

Comment:
http://lotsloki.api-docs.registry.ea2.openprocurement.io/en/latest/standard/item.html - this section has to be added instead of the given one